pennylane_lightning/lightning_qubit.py

```diff
@@ -16,6 +16,8 @@
 interfaces with C++ for fast linear algebra calculations.
 """
 from warnings import warn
+import os
```

Comment:
If I'm not mistaken the only use case is with `os.getenv` (like 311). So, 
```suggestion
from os import getenv
```

Reply:
Yep, good point, and well spotted.